ament_cmake_test/cmake/run_test.py

```diff
@@ -231,20 +246,30 @@ def log(msg, **kwargs):
     return rc
 
 
-def _generate_result(result_file, failure_message=None):
+def _generate_result(result_file, failure_message=None, skip=False):
```

Comment:
To avoid similar problems in the future the signature could be changed to:

```
def _generate_result(result_file, *, failure_message=None, skip=False):
```